test/quantization/test_quantize_script.py

```diff
@@ -1347,11 +1351,122 @@ def forward(self, x, y):
                  torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
         m = QuantizedAdd()
         m = self._test_op_impl(m, data, "quantized::add")
-        FileCheck().check_count("quantized::add", 3, exactly=True)
+        FileCheck().check_count("quantized::add", 3, exactly=True) \
+                   .run(m.graph)
         FileCheck().check_not("aten::add") \
                    .check_not("aten::add_") \
                    .run(m.graph)
 
+    @skipIfNoFBGEMM
```

Comment:
Why do we need to skip here? Should we just run over supported qengines?

Reply:
I can change this in a separate diff, fbgemm definitely works so I put it here.